lib/checks/navigation/heading-order.js

```diff
@@ -5,7 +5,7 @@ if (ariaHeadingLevel !== null) {
 	return true;
 }
 
-var headingLevel = node.tagName.match(/H(\d)/);
+var headingLevel = node.nodeName.match(/H(\d)/);
```

Comment:
do you need `toUpperCase()` here?

Reply:
I am going to leave this on as is (no need to uppercase).

Reply:
I disagree. In xhtml nodeName is lower case, which means this won't match.